deployments/liqo_chart/values.yaml

```diff
@@ -88,6 +88,9 @@ liqodash_chart:
     repository: "liqo/dashboard"
     pullpolicy: "Always"
   enabled: true
+  service: "liqo-dashboard"
```

Comment:
since you put these values for the clusterConfig, I would modify also the liqodash subchart (e.g. in the ServiceAccount template the name to use this value instead of the hard-coded one we now have, th same for the other values)